### PR TITLE
Feature/update local settings

### PR DESCRIPTION
**Pull request by jonsimonsen**
Get rid of some settings that are not interesting anymore (at least for people using ddev) given that sfp and dblog is reasonably set up in a dev split

### CODE REVIEW

**Comment by jonsimonsen** [comment]
@szeidler Could you review this and consider if it can be merged in? I think it would be nice if we would not have to remove large chunks of code from the example file when setting up local settings. If any of the deleted lines might still be of interest on some projects/setups, I'd suggest adding a separate file with the old version or some such into the scaffolding.

File: assets/settings.ramsalt.local.php [hunk 1]
```diff
@@ -29,19 +18,7 @@
 $settings['container_yamls'][] = 'sites/default/monolog.dblog.services.yml';
 
 // A default value for non-docker users' private files path.
```
Comment (szeidler) [inline assets/settings.ramsalt.local.php]: @jonsimonsen nitpick: The comment is not valid after the change anymore.

**Review by szeidler — CHANGES_REQUESTED** [review]

File: assets/settings.ramsalt.local.php (hunk 1 above)
Comment (jonsimonsen) [reply assets/settings.ramsalt.local.php]: I assume my latest commits fixes this. Feel free to merge it in if it's fine now

**Review by szeidler — APPROVED** [review]

File: assets/settings.ramsalt.local.php [hunk 2]
```diff
@@ -87,66 +53,3 @@
...
-}
-
-// Configure the database.
-$default_db_host      = 'mariadb';
-$default_db_port      = '3306';
-$default_db_name      = 'drupal';
-$default_db_user      = 'drupal';
-$default_db_password  = 'drupal';
-
-// Use .env database credential format if provided.
```
Comment (szeidler) [inline assets/settings.ramsalt.local.php:111]: The removal of the following lines has broken our automated tests. @jonsimonsen @jfauske we need to look into this.

Comment (jfauske) [reply assets/settings.ramsalt.local.php:111]: @szeidler Can you share an example of a failed job? Quickly looking at the tests job in the orb I don't immediately see where the issue arises,

Comment (szeidler) [reply assets/settings.ramsalt.local.php:111]: I already pushed a fix.

**Pull request merged**